packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/GenerateSummaryHandler.java

```diff
@@ -0,0 +1,16 @@
+package software.uncharted.terarium.hmiserver.service.tasks;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+// @Slf4j
+public class GenerateSummaryHandler extends TaskResponseHandler {
+	public static final String NAME = "gollm_task:generate_summary";
```

Comment:
pedantically, this can be `private` :D

Reply:
It looks like it needs to be public since it's used like `GenerateSummaryHandler.Name` without creating an instance of it in GoLLM Controller.